From 3ec8315a37ffc0634a81bf21a1cc4c1f2b3f4cfd Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 7 Sep 2018 13:15:20 -0600
Subject: [PATCH 3/5] klte-common: Remove irrelevant
 BOARD_HAVE_NEW_QCOM_CSDCLIENT flag

* This only exists in audio-caf/msm8960. It's not for us.

Change-Id: I5ddbbb6b45f42bd485ddb09619e918c74fb82369
---
 BoardConfigCommon.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index ad54467..a189611 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -21,7 +21,6 @@ COMMON_PATH := device/samsung/klte-common
 TARGET_SPECIFIC_HEADER_PATH := $(COMMON_PATH)/include
 
 # Audio
-BOARD_HAVE_NEW_QCOM_CSDCLIENT := true
 USE_CUSTOM_AUDIO_POLICY := 1
 
 # Bluetooth
-- 
2.17.1

